cmd2/cmd2.py

```diff
@@ -1942,13 +1873,13 @@ def preparse(self, raw):
         return raw
 
     # noinspection PyMethodMayBeStatic
-    def postparse(self, parse_result):
-        """Hook that runs immediately after parsing the command-line but before ``parsed()`` returns a ParsedString.
+    def postparse(self, statement):
```

Comment:
Recommend adding type hinting for new/modified code.

Reply:
Fixed and pushed to `ply` branch.